commit 7de3f0a96c4e93a7cd7312296d00039604ed94f8
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Tue Sep 27 23:37:42 2022 +0200

    hurd: Increase SOMAXCONN to 4096
    
    Notably fakeroot-tcp may introduce a lot of parallel connections.

diff --git a/sysdeps/mach/hurd/bits/socket.h b/sysdeps/mach/hurd/bits/socket.h
index 70fce4fb27..4ece37f1dd 100644
--- a/sysdeps/mach/hurd/bits/socket.h
+++ b/sysdeps/mach/hurd/bits/socket.h
@@ -147,7 +147,7 @@ enum __socket_type
 #define	AF_MAX		PF_MAX
 
 /* Maximum queue length specifiable by listen.  */
-#define SOMAXCONN	128	/* 5 on the origional 4.4 BSD.  */
+#define SOMAXCONN	4096	/* 5 on the origional 4.4 BSD.  */
 
 /* Get the definition of the macro to define the common sockaddr members.  */
 #include <bits/sockaddr.h>
